metadata.rb

```diff
@@ -16,3 +16,5 @@
 end
 
 depends 'line'
+depends 'sysctl'
+depends 'chef-sugar'
```

Comment:
Looks like I used rhel? in the server recipe. We use chef-sugar for things internally, so I've gotten accustomed to some of the things just always being available. Happy to change it if you'd rather remove the extra dependency.